scripts/zones/Windurst_Waters/npcs/Leepe-Hoppe.lua

```diff
@@ -81,9 +100,14 @@ function onTrigger(player,npc)
             player:startEvent(850,0,13399,1208,1125,availRewards,18165,13572);
         elseif (realday ~= player:getCharVar("MoonlitPath_date")) then --24 hours have passed, flag a new fight
             player:startEvent(848,0,1125,334);
-        else
-            player:startEvent(847,0,1125); -- Yes, this will indefinitely replace his standard dialogue!
         end
+
+    elseif tuningIn == QUEST_ACCEPTED then
+        player:startEvent(885, 0, 1696, 1697, 1698) -- Reminder to bring Magicked Steel Ingot, Spruce Lumber, Extra-fine File
+
+    elseif moonlitPath == QUEST_COMPLETED then
+        player:startEvent(847,0,1125) -- Having completed Moonlit Path, this will indefinitely replace his standard dialogue!
```

Comment:
Can you space the params as you did rest please

Reply:
Sure thing. This was just moved down from a few lines above.